samples/event_manager_profiler/README.rst

```diff
@@ -0,0 +1,77 @@
+.. _event_manager_profiling_sample:
+
+Event Manager profiling tracer
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Event Manager profiling tracer sample demonstrates the functionality of profiling :ref:`event_manager` events using the :ref:`profiler` and the Event Manager profiling tracer modules.
```

Comment:
I would add reference to Event Manager Profiler tracer instead of profiler.

Reply:
Will add as soon as documentation for it will be merged.

Reply:
Keep in mind to add the ref